### PR TITLE
param tests failing

### PR DESCRIPTION
**Pull request by genaroNF**
The tests at test_GtagMP_internal_param_functions.py were failing, this  PR fixes them by adding the necessary functions

didn't wanted to delete anything in case it was going to be used in the future

### CODE REVIEW

**Comment by genaroNF** [comment]
Hey, @rptomey I think you added this, could you take a look?

File: ga4mp/store.py [hunk 1]
```diff
@@ -21,6 +21,11 @@ def _set(self, param_type, name, value):
         # Helper function to set a single parameter (user or session or other).
         self._check_exists(key=param_type)
         self[param_type][name] = value
+    
+    def _delete(self, param_type, name):
+        # Helper function to set a single parameter (user or session or other).
```
Comment (rptomey) [inline ga4mp/store.py:26]: Comment should be updated to accurately describe the function.

File: tests/test_GtagMP_internal_property_functions.py [hunk 2]
```diff
@@ -26,23 +26,23 @@ def setUp(self):
...
-        self.gtag.set_user_property("user_id","Grogu")
+        self.gtag.store.set_user_property("user_id","Grogu")
         
-        self.assertEqual(self.gtag._user_properties["user_id"], "Grogu")
+        self.assertEqual(self.gtag.store["user_properties"]["user_id"], "Grogu")
 
-    def test_add_multiple_user_properties(self):
-        self.gtag.set_user_property("user_id","Grogu")
-        self.gtag.set_user_property("region","Outer Rim")
+    def test_add_multiplestore(self):
```
Comment (rptomey) [inline tests/test_GtagMP_internal_property_functions.py:33]: I'd prefer to not change the name for this method.

**Review by rptomey — APPROVED** [review]
Code works and addresses some issues. I have a couple of minor nitpicks, but those can be addressed after merging to the `dev` branch.

**Pull request merged**